### PR TITLE
Update registry.json with FAM

### PR DESCRIPTION
**Pull request by darkartopiafam**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3350,5 +3350,20 @@
...
+       "twitter":"darkartopia",
+       "website":"http://darkartopia.art",
+       "reddit":"",
+       "instagram":"darkartopia",
+       "youtube":""
+    },
+    "telegram":"darkartopia",
+    "operatorName":"Marianna",
+    "github":"darkartopiafam"
+  },
```
Comment (Bez625) [inline registry.json:3368]: Unfortunately a trailing comma is not allowed in JSON. Please can you remove?

**Review by Bez625 — CHANGES_REQUESTED** [review]
Please remove comma

**Comment by darkartopiafam** [comment]
I should have been more careful. I'm sorry

**Comment by Bez625** [comment]
> I should have been more careful. I'm sorry

Haha no need to be sorry!

**Pull request merged**